### DISCUSSION TITLE
Option for miles and miles/kWh units

### DISCUSSION THREAD

**Discussion by sOckhamSter** [flobz/psa_car_controller#324] [Ideas]
Would it be possible to have a configuration toggle to switch between displaying kilometres and miles? My brain doesn't work well looking at consumption values in kWh/100km and being able to view this data on the Trips list in miles/kWh would be a lot easier for me to read. My car (Corsa-e) also displays the consumption in miles/kWh so it would help it match up nicely.

**Comment by flobz** [comment]
A British developer might do it one day.
